### PR TITLE
Bug 278

### LINKED ISSUE #278: Redis constructor type does not fully express its valid uses
[`@keyv/redis`'s constructor signature](https://github.com/jaredwray/keyv/blob/main/packages/redis/src/index.d.ts#L7) does not support passing it a ` Redis.Cluster` or `Redis` instance which looks like it's supported based on the [implementation](https://github.com/jaredwray/keyv/blob/main/packages/redis/src/index.js#L11)

`@keyv/redis` version `2.3.1`

### PR DESCRIPTION
**Pull request by alphmth**
Fixes #278

### CODE REVIEW

**Comment by alphmth** [comment]
@jaredwray Removed this happens during merging

File: packages/redis/src/index.d.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ declare class KeyvRedis extends EventEmitter implements Store<string | undefined
 	has(key: string): boolean;
 }
 declare namespace KeyvRedis {
-	interface Options extends ClientOpts {
+	interface Options {
```
Comment (gnarea) [inline packages/redis/src/index.d.ts:22]: @jaredwray, I believe `@keyv/redis@2.3.3` is supposed to have this change, but it doesn't:

```
> tsc -p tsconfig.json

node_modules/@keyv/redis/src/index.d.ts:22:28 - error TS2304: Cannot find name 'ClientOpts'.

22  interface Options extends ClientOpts {
                              ~~~~~~~~~~


Found 1 error in node_modules/@keyv/redis/src/index.d.ts:22



$ grep @keyv/redis package.json 
    "@keyv/redis": "^2.3.3",
```

**Comment by jaredwray** [comment]
@gnarea - can you do this as a new pull request and I can get this validated and updated?

**Comment by gnarea** [comment]
@jaredwray, `main` already has the correct code:

https://github.com/jaredwray/keyv/blob/05e312bed01b4f08ce06e215d3a557df2e41184b/packages/redis/src/index.d.ts#L22

I wonder if this is a packaging/release issue?

**Comment by jaredwray** [comment]
@gnarea Might be. Will validate and look to release today.

**Comment by jaredwray** [comment]
@gnarea - @keyv/redis version `2.3.4` has been released with the correct `index.d.ts` file

**Pull request merged**